app/models/driver.rb

```diff
@@ -0,0 +1,30 @@
+class Driver < ApplicationRecord
+  validates_presence_of :vin
+  validates :name, presence: true, format: { with:/\A\S+\s{1}.+\z/ , message: "Please enter a first and last name."}
+
```

Comment:
Good set fo validations and business logic methods, consider using enumerables next project.